tests/unit/test.spec.js

```diff
@@ -0,0 +1,36 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+import AddMemorialForm from '@/components/AddMemorialForm.vue'
+import VueI18n from 'vue-i18n'
+
+describe("addmemorial testl", () => {
+    let localVue = createLocalVue()
+    localVue.use(VueI18n)
+    test = {
```

Comment:
usually when you have an object like this that you're using as a strict equals, you want to call it a "golden" object